src/frontend/src/components/public/CommentsSidebar/index.tsx

```diff
@@ -27,6 +30,12 @@ const CommentSidebar = ({
   const [modalTitleText, setModalTitleText] = useState<string>('');
   const [modalParagraph, setModalParagraph] = useState<string>('');
   const [modalButtonValue, setModalButtonValue] = useState<string>('');
+  const [enableComments, setEnableComments] = useState<boolean>(true);
```

Comment:
I would advice disabling comments by default. Let me know if there is a specific reason you are thinking otherwise?